packages/utilities/src/utilities/sameheight/sameheight.js

```diff
@@ -20,7 +20,7 @@
  * @param {string} minSize Minimum size for the utility to be activated, empty for small,
  *  md for medium, lg for large, xlg for xlarge, max for maximum
  */
-function sameheight(elemCollection, minSize) {
+function sameHeight(elemCollection, minSize = false) {
   const elemArr = Array.prototype.slice.call(elemCollection);
   let targetWidth = 0;
   if (minSize) {
```

Comment:
I'm looking at lines `26-41`, and I'm a little nervous we've hard coded the breakpoints in here. Can we leverage the `@carbon/layout` package to get the proper breakpoints from the source?

https://github.com/carbon-design-system/carbon/blob/master/packages/layout/src/index.js#L45-L71

Reply:
If I'm understanding what you're doing here you could just do something along the lines of the following...

```javascript
import { breakpoints } from '@carbon/layouts';

function sameheight(elemCollection, minSize) {
  const target = breakpoints[minSize];
  const targetWidth = target ? parseFloat(target.width) * baseFontSize : 0;

  ...
}
```

`parseFloat()` and `baseFontSize` takes `1rem` and converts to `16` for our needs.